native/python/pyjp_value.cpp

```diff
@@ -169,6 +169,9 @@ PyObject* PyJPValue_str(PyObject* self)
 	if (cls->isPrimitive())
 		JP_RAISE(PyExc_TypeError, "toString requires a Java object");
 
+	if (value->getValue().l == NULL)
+		return JPPyString::fromStringUTF8("null").keep();
+
```

Comment:
Nulls now covert to "null" when calling string rather than value error.

Reply:
Why would a mapping to an empty string be a bad idea in this case? I mean I'm fine with the "null" string too, but mapping nothing to a real word seems somehow odd. Is it a Java convention to do this mapping?

Reply:
The problem is if you don't make it "null" then you get a ValueError when you are printing.

Consider the following Java code
``` java
 public static void main(String[] args)
  {
    Object o = null;
    System.out.println("A " + o);
  }
```

The correct output is "A null".   So to str(o) is "null".

Reply:
I see.